docs/.vuepress/config.js

```diff
@@ -378,25 +378,32 @@ module.exports = {
   port: 8080,
   description: 'The headless CMS developers love.',
   base: '/documentation/',
-  plugins: {
-    '@vuepress/medium-zoom': {},
-    'vuepress-plugin-element-tabs': {},
-    '@vuepress/google-analytics': {
+  plugins: [
+    ['vuepress-plugin-element-tabs', {}],
+    ['@vuepress/google-analytics', {
       ga: 'UA-54313258-1',
-    },
-    'check-md': {
+    }],
+    ['check-md', {
       ignore: checklinksIgnoredFiles,
-    },
-    seo: {
+    }],
+    ['seo', {
       siteTitle: (_, $site) => $site.title,
       title: $page => $page.title,
-    },
-    'vuepress-plugin-code-copy': {
+    }],
+    ['vuepress-plugin-code-copy', {
       color: '#ffffff',
       successText: 'Copied to clipboard!',
-    },
-    '@vuepress/back-to-top': {},
-  },
+    }],
+    ['@vuepress/back-to-top', {}],
+    ['vuepress-plugin-container', {
+      type: 'strapi',
+      defaultTitle: '🤓 '
+    }],
+    ['vuepress-plugin-container', {
+      type: 'prerequisites',
```

Comment:
Wonder if we should generalize the name :thinking: instead of using `prerequisites` since we use it elsewhere besides just prerequisites.

Though if plan to add a bunch (would be nice) we probably need to document them somewhere, not needed for now but just a thought

Reply:
That's a good point, thank you!

What about:
* keeping this one (so we have the "PREREQUISITES" defaultTitle ready)
* and create a 'generic' call-out, with the same look but without the defaultTitle?

I've just started documenting them [in Notion](https://www.notion.so/strapi/22bb07756e954ad6b15e429ba3674f51?v=5fd2c112ccbe40e38138d0a1d4f60c00) 🙂

Reply:
I've updated the PR description to include these changes :-)